packages/components/modals/ModalWithoutHeader.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import ModalBase, { ModalBaseProps } from "./ModalBase";
+
+interface ModalWithoutHeaderProps extends ModalBaseProps {}
+
+const ModalWithoutHeader: React.FC<ModalWithoutHeaderProps> = ({
```

Comment:
Don't default export the component, write `export const ...`

Reply:
OK, done